scripts/docker_push.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+GIT_COMMIT_ID_SHORT=`git rev-parse --short HEAD`
+if [[ -n `git status --porcelain --untracked-files=no` ]]; then
+    # uncommitted changes - dirty
+    GIT_COMMIT_ID_SHORT="$GIT_COMMIT_ID_SHORT-dirty"
+fi
+
+IMAGE_NAME="quay.io/codeready-toolchain/registration-service:$GIT_COMMIT_ID_SHORT"
```

Comment:
what about using `quay.io/${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}` to be consistent with what's in `make/dev.mk` (https://github.com/codeready-toolchain/registration-service/pull/40/files#diff-fd8f90289cf39bfe347fc71f872e5861R72)?

Reply:
Because I was too lazy to calculate org and repo names in docker_push.sh (we can't reuse our make here). Also docker_push.sh is purely for travis, so, I think it's OK to keep it simpler for now.

Reply:
ok, fair point